services/ipnhandler.php

```diff
@@ -358,6 +358,8 @@
 //Other
 //if we got here, this is a different kind of txn
 ipnlog( "No recurring payment id or item number. txn_type = " . $txn_type );
+
+do_action( 'pmpro_webhook_unhandled' );
```

Comment:
Let's add hook doc info via phpdoc for new actions.

```suggestion
/**
 * Allow hooking into after a webhook has been run but was not handled.
 *
 * @since TBD
 *
 * @param string $gateway The gateway the webhook was not handled for.
 */
do_action( 'pmpro_webhook_unhandled', PMPRO_DOING_WEBHOOK );
```